codalab/lib/bundle_cli.py

```diff
@@ -1237,7 +1237,7 @@ def _worksheet_description(self, worksheet_info):
         fields = [
             ('Worksheet', self.worksheet_str(worksheet_info)),
             ('Title', formatting.verbose_contents_str(worksheet_info['title'])),
-            ('Tags', worksheet_info['tags']),
+            ('Tags', ', '.join(worksheet_info['tags'])),
```

Comment:
Are you saying that keeping tags to single-words is not enforced? I can issue that